.gitignore

```diff
@@ -1,2 +1,4 @@
 /lex.yy.c
 /cvsclone
+
+*.dSYM
```

Comment:
**.dSYM** is a debug symbol file thats output when I build on OS X. It's not needed and I don't want it checked in by accident.